tests/console/check_default_network_manager.pm

```diff
@@ -26,7 +26,8 @@ sub run {
         zypper_call 'in systemd-network';
         systemctl 'is-enabled systemd-networkd', expect_false => 1;
         systemctl 'is-active systemd-networkd', expect_false => 1;
-        assert_script_run 'networkctl status';
+        script_run 'networkctl status';
```

Comment:
Do you see any value of keeping it for debugging purposes?

Reply:
> Do you see any value of keeping it for debugging purposes?

Yes. It doesn't hurt.

Reply:
it is just redundant :shrug: as the output of both is identical

Reply:
> it is just redundant 🤷 as the output of both is identical

The output is different on all systems I tried it.

Reply:
The message that networkd is not running is in both, anyway. It doesn't hurt unless it would end up in a timeout. but that is another thing